### PR TITLE
Update main README file

### PR DESCRIPTION
**Pull request by manbearshark**
*Issue #, if available:*

The main README file was in need of an update to support all of the different use cases for the repo.

*Description of changes:*

- Added an overview of the workshops included in the repo
- Cleaned up and reorganized deployment instructions for general users and developers

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,44 +2,70 @@
...
 ![Retail Demo Store Architecture](./workshop/images/retaildemostore-architecture.png)
 
-## Workshops
+## Hands-On Workshops
+
+This project is designed to provide you with an environment in which you can learn to use AWS services to modify the behavior of an ecommerce application, based on business requirements. This can be done in a group setting or as an individual using self-paced workbooks. Currently there are workshops for adding search, personalization, experimentation frameworks, a/b testing, analytics, CDPs, messaging, and more. 
+
+In order to use the workshops, you will need to deploy the Retail Demo Store into an AWS account, using one of the methods described in the Getting Started or Developers sections below.  This is necessary because the workshops run in SageMaker Jupyter notebooks, which provide an interactive Python environment where you can execute code in the Retail Demo Store environment.
+
+AWS Service | Workshops Overview | Workshop Links
```
Comment (BastLeblanc) [inline README.md:23]: why skipped the Experimentation one? 

Maybe add a difficulty level (search is probably 100/200, experimentation 400)

**Review by BastLeblanc — COMMENTED** [review]
The README is becoming massive, can we try to break it down to sub sections
- intro / delivering a demo : README
- workshops: README (or side file)
- dev : How to contribute
- Deploy : HOWTODEPLOY

it's a suggestion, happy to discuss.

File: README.md [hunk 2]
```diff
@@ -2,44 +2,70 @@
...
+--- | --- | ---
+![Amazon Personalize](./workshop/images/Amazon-Personalize_64.png) | The Retail Demo Store uses Amazon Personalize to provide similar item recommendations, search re-ranking based on user preferences, and product recommendations based on user item interactions.  The attached workshop is a throrough walk through of the major features of Amazon Personalize, and how it can be deployed in an ecommerce application like the Retail Demo Store. | [Personalize Setup](./workshop/1-Personalization/1.1-Personalize.ipynb)
```
Comment (BastLeblanc) [inline README.md:25]: I would add the service name under each logo

File: README.md [hunk 3]
```diff
@@ -48,63 +74,144 @@ Europe (Ireland) | eu-west-1 | [![Launch Stack](https://cdn.rawgit.com/buildkite
...
+[Detailed instructions](./src) are available on how to get going with local development.  Note that you will still need to set up one of the development options described above, and have a working deployment in your AWS account as some of the services will need to access cloud-based services as part of deployment.
+
+# Delivering a Demo of the Retail Demo Store
+
+Once you have deployed the Retail Demo Store, you may want to walk through the demonstration guide to learn how to show the features the Retail Demo Store provides.
+
+The intent of the Retail Demo Store is to 1) provide a tool to demonstrate the capabilities of key AWS services for retail, eCommerce, and digital marketing use-cases and 2) provide a platform for individual AWS customers to step through workshops and AWS internal teams to deliver customer-facing workshops, Immersion Days, hackathons, and similar types of events.
+
+1) [Creating a Retail Demo Store account](1-Creating-account.md)
+2) [Personalized Experience](2-Personalization.md)
```
Comment (james-jory) [inline README.md:202]: These links are broken. Should path into "documentation".

**Review by james-jory — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (manbearshark) [reply README.md:25]: Great suggestions.  We could do both - though there might need to be some tweaking of the table sizing due to the font sizing on GitHub.  Would need to play around with this a bit so it does not look wonky.

File: README.md [hunk 4]
```diff
@@ -2,44 +2,70 @@
...
+
+At the heart of the Retail Demo Store is a collection of polyglot microservices hosted in Amazon Elastic Container Service (AWS Fargate) that represent domain constructs such as [products](../src/products), [carts](../src/carts), [orders](../src/orders), and [users](../src/users) as well as services for [search](../src/search) and [recommendations](../src/recommendations). While the [web user interface](../src/web-ui) is served by Amazon CloudFront and Amazon S3.
+
+The architecture is supported by several managed services including Amazon Cognito, Amazon Pinpoint, Amazon Personalize, and Amazon Elasticsearch. The [web user interface](../src/web-ui]) is built using the Vue.js framework with AWS Amplify to provide integrations with Cognito for registration/authentication and event streaming to Pinpoint and Personalize (Event Tracker). Finally, AWS CodePipeline is leveraged to demonstrate how AWS development services can be used to orchestrate the build and deployment process with the Retail Demo Store.
```
[the remark this answers is not in the document]
Comment (manbearshark) [reply README.md:13]: Will do.

File: README.md [hunk 5]
```diff
@@ -2,44 +2,70 @@
...
 
 ![Retail Demo Store Architecture](./workshop/images/retaildemostore-architecture.png)
 
-## Workshops
+## Hands-On Workshops
+
+This project is designed to provide you with an environment in which you can learn to use AWS services to modify the behavior of an ecommerce application, based on business requirements. This can be done in a group setting or as an individual using self-paced workbooks. Currently there are workshops for adding search, personalization, experimentation frameworks, a/b testing, analytics, CDPs, messaging, and more.
```
[the remark this answers is not in the document]
Comment (manbearshark) [reply README.md:19]: This is a very good point.

**Comment by james-jory** [comment]
Thinking about @BastLeblanc's comment on the length of the README a bit more, I think moving the "Developer Instructions" and perhaps the "Delivering a Demo fo the RDS" to another page makes sense.

File: README.md (hunk 3 above)
Comment (manbearshark) [reply README.md:202]: Good catch.  Will fix.

Comment (james-jory) [reply README.md]: Fixed

File: README.md [hunk 6]
```diff
@@ -2,44 +2,70 @@
...
 
 ![Retail Demo Store Architecture](./workshop/images/retaildemostore-architecture.png)
 
-## Workshops
+## Hands-On Workshops
+
+This project is designed to provide you with an environment in which you can learn to use AWS services to modify the behavior of an ecommerce application, based on business requirements. This can be done in a group setting or as an individual using self-paced workbooks. Currently there are workshops for adding search, personalization, experimentation frameworks, a/b testing, analytics, CDPs, messaging, and more. 
+
+In order to use the workshops, you will need to deploy the Retail Demo Store into an AWS account, using one of the methods described in the Getting Started or Developers sections below.  This is necessary because the workshops run in SageMaker Jupyter notebooks, which provide an interactive Python environment where you can execute code in the Retail Demo Store environment.
```
[the remark this answers is not in the document]
Comment (james-jory) [reply README.md:21]: Added a row for the Experimentation workshops and columns for level and duration.

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (james-jory) [reply README.md]: Changed

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (james-jory) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (james-jory) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (james-jory) [reply README.md]: Done

**Comment by james-jory** [comment]
Moved developer instructions to a separate page to reduce weight of main README.

**Review by james-jory — APPROVED** [review]

**Pull request merged**